pygmt/clib/session.py

```diff
@@ -112,7 +112,7 @@ class Session:
     """
 
     # The minimum version of GMT required
-    required_version = "6.0.0rc4"
+    required_version = "6.0.0rc2"
```

Comment:
I probably shouldn't have committed this

Reply:
I was just looking at this too. You should be able to revert that commit easily enough though.